src/app/components/search-field/search-field.component.html

```diff
@@ -0,0 +1,13 @@
+<onto-search-field
+  [autocompleteData]="states"
+  [customTemplate]="useCustomTemplate() ? customTemplate : null"
+  [selectedList]="getSelectedList()"
+  [enableInnerAutocompleteFiltration]="enableInnerAutocompleteFiltration()"
+  (onSearch)="onSearchEvent()"
+  (onKeyPress)="onKeyPressEvent()"></onto-search-field>
+
+<ng-template #customTemplate let-item>
```

Comment:
This looks like the default template.

Reply:
with different selectors only - IMO it's enough for testing